### PR TITLE
Toast rating

### LINKED ISSUE #7: Snackbar/Toast/Notify
image

### LINKED ISSUE #21: Rating
image

### PR DESCRIPTION
**Pull request by mohitdhatrak**
Deploy preview: https://deploy-preview-23--spark-ui.netlify.app/

# Toast

image

# Rating

image

On hover effect -

image

Closes #7 
Closes #21

### CODE REVIEW

File: Components/Rating/rating.html [hunk 1]
```diff
@@ -0,0 +1,254 @@
+<!DOCTYPE html>
+<html lang="en">
+    <head>
+        <meta charset="UTF-8" />
+        <meta http-equiv="X-UA-Compatible" content="IE=edge" />
```
Comment (Rishi05051997) [inline Components/Rating/rating.html:5]: Please include meta tag like author name & description

File: Components/Rating/rating.css [hunk 2]
```diff
@@ -0,0 +1,59 @@
+* {
+    box-sizing: border-box;
```
Comment (bharati-21) [inline Components/Rating/rating.css:4]: Maybe create a separate CSS file called reset.css for resetting default styles.

**Review by Rishi05051997 — COMMENTED** [review]
Suggested minor change then go to go

File: Components/Rating/rating.html [hunk 3]
```diff
@@ -0,0 +1,254 @@
...
+        <meta name="viewport" content="width=device-width, initial-scale=1.0" />
```
Comment (bharati-21) [inline Components/Rating/rating.html:6]: Suggest adding meta author and description tag.

File: Components/Rating/rating.html [hunk 4]
```diff
@@ -0,0 +1,254 @@
...
+                        <a
+                            href="/Components/Toast/toast.html"
+                            class="drawer-link"
+                            >Toast</a
+                        >
+                    </div>
+                </aside>
+            </div>
+
+            <div>
```
Comment (bharati-21) [inline Components/Rating/rating.html]: Suggestion to change divs to sematic tags.

File: Components/Rating/rating.html [hunk 5]
```diff
@@ -0,0 +1,254 @@
...
+                        <span class="code-highlight">:root</span> selector of
+                        your CSS file (unique variables for both types so you
+                        can use both together too). No need to use these
+                        variables if you wish to use the component as it is. For
+                        the star icons to be visible, you need to link the given
+                        google-icons stylesheet in your HTML too.
+                    </p>
+                    <iframe
+                        src="https://www.thiscodeworks.com/embed/6200ccbe4dd44f0015f3cd70"
+                        style="height: 17.85rem"
```
Comment (bharati-21) [inline Components/Rating/rating.html:179]: Kindly avoid inline styles.

File: Components/Toast/toast.css [hunk 6]
```diff
@@ -0,0 +1,103 @@
...
+    align-items: center;
+    gap: 1.5em;
+}
+
+.toast.toast-left {
+    right: none;
+    left: 1.2rem;
+}
+
+@media screen and (max-width: 1030px) {
```
Comment (bharati-21) [inline Components/Toast/toast.css:40]: Another suggestion - to club media queries and add it towards the end of the sheet.

File: Components/Toast/toast.html [hunk 7]
```diff
@@ -0,0 +1,232 @@
...
+        <link
+            rel="shortcut icon"
+            href="/Images/spark-ui-favicon.png"
+            type="image/png"
+        />
+        <link rel="stylesheet" href="/Components/Toast/toast.css" />
+        <link rel="stylesheet" href="/main.css" />
+        <link
+            rel="stylesheet"
+            href="https://fonts.googleapis.com/icon?family=Material+Icons"
```
Comment (bharati-21) [inline Components/Toast/toast.html]: Please import the material icon link through CSS.

**Review by bharati-21 — APPROVED** [review]
Looks good to me. Great choice of colors and font. Just left a few suggestions you could consider.

File: Components/Rating/rating.html (hunk 1 above)
Comment (mohitdhatrak) [reply Components/Rating/rating.html:5]: Ok will do that after deadline! Have less time now

File: Components/Toast/toast.html (hunk 7 above)
Comment (mohitdhatrak) [reply Components/Toast/toast.html]: Thanks for the suggestion!

File: Components/Rating/rating.html (hunk 4 above)
Comment (mohitdhatrak) [reply Components/Rating/rating.html]: Done used main, section, article, aside, nav tags

File: Components/Toast/toast.html (hunk 7 above)
Comment (mohitdhatrak) [reply Components/Toast/toast.html]: Done

File: Components/Rating/rating.html (hunk 1 above)
Comment (mohitdhatrak) [reply Components/Rating/rating.html:5]: Done

**Pull request merged**